impl/gs_audio_impl.h

```diff
@@ -21,8 +22,8 @@
 
 // Includes
 #include "../external/stb/stb_vorbis.c"
-#include "../external/dr_libs/dr_wav.h"
-#include "../external/dr_libs/dr_mp3.h"
+#define MINIAUDIO_IMPLEMENTATION
```

Comment:
Why is this moved outside of the miniaudio gs impl define? It needs to be encapsulated in that or else miniaudio will get implemented in all audio implementations.